html/admin/asset.php

```diff
@@ -129,6 +129,51 @@ function linkedAssets($assetId,$tier) {
         $DBLIB->where("assetGroups_id IN (" . $PAGEDATA['assets'][0]['assets_assetGroups'] . ")");
         $PAGEDATA['assets'][0]['groups'] = $DBLIB->get("assetGroups",null,["assetGroups_id","assetGroups_name"]);
     } else $PAGEDATA['assets'][0]['groups'] = [];
+
+    //Asset Location
+    //Current asset location
+    $latestScan = assetLatestScan($asset['assets_id']);
+
+    if(isset($latestScan['locations_id'])) {
+        $PAGEDATA['assets'][0]['assets_storageLocationName'] = $latestScan['locations_name'];
```

Comment:
Could we tweak this var name? Sorry being a bit pedantic but I can see us implementing something like that another time

Reply:
Good shout